list.txt

```diff
@@ -13964,7 +13927,6 @@ hooverexpress.net
 hop2.xyz
 hopemail.biz
 hopoverview.com
-hopto.org
```

Comment:
and this one

Reply:
add new commit - where SLD domain allowed